src/globus_cli/parsing/commands.py

```diff
@@ -108,8 +123,9 @@ def command(*args, **kwargs):
     def _inner_decorator(func):
         if "help" not in kwargs:
             kwargs["help"] = func.__doc__
-
         if "cls" not in kwargs:
+            kwargs["cls"] = GlobusCommandEnvChecks
+        if kwargs.get("skip_env_checks", False) is True:
```

Comment:
I don't know that we use the `cls` argument anywhere, but this makes `cls=Foo, skip_env_checks=True` behave surprisingly.

Reply:
Fair point. I updated to (I think) a more sensible behavior, and added a note.